ios/Classes/BleData extras/QualifiedCharacteristic.swift

```diff
@@ -13,8 +13,8 @@ extension QualifiedCharacteristic {
     init(_ characteristic: CBCharacteristic) {
         self.init(
             id: characteristic.uuid,
-            serviceID: characteristic.service.uuid,
-            peripheralID: characteristic.service.peripheral.identifier
+            serviceID: characteristic.service!.uuid,
+            peripheralID: characteristic.service!.peripheral!.identifier
```

Comment:
Optionality was matching before, but not anymore? Please, comment on every case (why exactly? why not guard-throw or init?-null)

Reply:
As I mentioned Swift is a language I never really use. I unwrapped the variables with a `!` just to fix the problem. I tested it and it worked. I don't have enough Swift / native iOS development experience to properly solve the problem. This is something someone else would have to look into with more experience.

Reply:
Clear. I appreciate your effort any way, but this is something to be clarified and refined before we merge this p/r.

I'll try to find time to look into this in the coming week or two.